lib/commands/general.js

```diff
@@ -27,21 +27,46 @@ commands.doSendKeys = async function (params) {
 
 /**
  * Get device time
- * @param {String} format A optional format for `adb shell date`. You can see help with `adb shell date --help`.
- *                        `iso-8601`format by default like `2018-06-09T16:21:54+0900`.
- *                        e.g.: `'+%Y-%m-%dT%T%z'`
+ * @param {String} format A optional format for `adb shell date`.
+ *                        `iso-8601`format is default like `2018-06-09T16:21:54+0900` by `'+%Y-%m-%dT%T%z'`
+ *
+ *                        usage: date [-u] [-r FILE] [-d DATE] [+DISPLAY_FORMAT] [-D SET_FORMAT] [SET]
+ *                        Set/get the current date/time. With no SET shows the current date.
+ *
+ *                        Default SET format is "MMDDhhmm[[CC]YY][.ss]", that's (2 digits each)
+ *                        month, day, hour (0-23), and minute. Optionally century, year, and second.
+ *                        Also accepts "@UNIXTIME[.FRACTION]" as seconds since midnight Jan 1 1970.
+ *
+ *                        -d	Show DATE instead of current time (convert date format)
+ *                        -D	+FORMAT for SET or -d (instead of MMDDhhmm[[CC]YY][.ss])
+ *                        -r	Use modification time of FILE instead of current date
+ *                        -u	Use UTC instead of current timezone
+ *
+ *                        +FORMAT specifies display format string using these escapes:
```

Comment:
Only keep the stuff below. The list of available date arguments is not needed here.